Tasks/SSH/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "ssh2": "0.5.0",
+    "vsts-task-lib": "0.8.2"
```

Comment:
This needs to be updated to 0.8.3 after the task-lib changes are merged: https://github.com/Microsoft/vsts-task-lib/pull/91